src/data/javascript-quiz.ts

```diff
@@ -2084,6 +2084,36 @@ const javascriptQuiz = [
     Explanation:
       "Variable names cannot start with a number or any special character.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types"
+  },
+  {
+    Question: "If const a = 2 and then a = 5 what will be the result?",
+    Answer: "A TypeError ",
+    Distractor1: "A TypeError ",
+    Distractor2: "5",
+    Distractor3: "2",
+    Explanation:
+      "In JavaScript, a variable declared with the `const` keyword cannot be reassigned.",
```

Comment:
we should remove the backticks from the const keyword.
Otherwise the backticks will show up on the site.

```suggestion
      "In JavaScript, a variable declared with the const keyword cannot be reassigned.",
```